test/elixir_analyzer/test_suite/dancing_dots_test.exs

```diff
@@ -0,0 +1,270 @@
+defmodule ElixirAnalyzer.TestSuite.DancingDotsTest do
+  use ElixirAnalyzer.ExerciseTestCase,
+    exercise_test_module: ElixirAnalyzer.TestSuite.DancingDots
+
+  test_exercise_analysis "example solution",
+    comments: [ElixirAnalyzer.Constants.solution_same_as_exemplar()] do
+    ~S"""
```

Comment:
I didn't know we couldn't define more than one module at a time... Is that worth looking into?

Reply:
Yeah, it doesn't work. I assume it would be possible to find the right addition to this snippet that would make it possible: https://github.com/exercism/elixir-analyzer/blob/0a1bd66587808f72cec0ef7404fc6708da0058ad/test/support/exercise_test_case.ex#L86-L90

But I don't feel a strong need to work on it at the moment 😇